README.md

```diff
@@ -68,6 +68,19 @@ asdf global deno 0.2.10
 asdf local deno 0.2.10
 ```
 
+**With [Scoop](https://scoop.sh):**
```

Comment:
Duplicated link. Maybe link here https://github.com/lukesampson/scoop/wiki/Switching-Ruby-And-Python-Versions instead?

Reply:
agree